yarn.lock

```diff
@@ -14165,11 +14165,16 @@ focus-lock@^0.7.0:
   resolved "https://registry.yarnpkg.com/focus-lock/-/focus-lock-0.7.0.tgz#b2bfb0ca7beacc8710a1ff74275fe0dc60a1d88a"
   integrity sha512-LI7v2mH02R55SekHYdv9pRHR9RajVNyIJ2N5IEkWbg7FT5ZmJ9Hw4mWxHeEUcd+dJo0QmzztHvDvWcc7prVFsw==
 
-follow-redirects@^1.0.0, follow-redirects@^1.15.0:
+follow-redirects@^1.0.0:
   version "1.15.2"
   resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.15.2.tgz#b460864144ba63f2681096f274c4e57026da2c13"
   integrity sha512-VQLG33o04KaQ8uYi2tVNbdrWp1QWxNNea+nmIB4EVM28v0hmP17z7aG1+wAkNzVq4KeXTq3221ye5qTJP91JwA==
 
+follow-redirects@^1.15.6:
```

Comment:
This one ^ is probably the keeper

Reply:
Don't think so, rebuilt it a couple of times, it doesn't get deduplicated

Reply:
You can force these by adding a `"resolutions"` entry to the `package.json` or manually editing the package-lock. 

The question is, why do these need to be consolidated, @kc13greiner?

Reply:
I see, because the other versions are claimed to be vulnerable

Reply:
> manually editing the package-lock
This might have issues, especially on different platforms

Reply:
You can add to the package.json's `"resolutions"` field, re-run `yarn kbn bootstrap`, and it would be consolidated.
```
    "**/follow-redirects": "^1.15.6",
```

Reply:
already pushed it in 2152878eb1f2041edb9e275b9b1b0e53dad6506f

Reply:
We want to avoid using "resolutions" where possible since that will lock us into potentially outdated versions.

I see we had `axios` set to 1.6.3, now it has been updated to the "latest" (1.7.5 came out 4 days ago) is there any reason to pin these at this point?

Reply:
@delanni 

> The question is, why do these need to be consolidated

Ideally we would like to have only the latest versions of deps (or sub deps), especially if they evaluate to the same semver.

We are happy to provide more information over at the #kibana-security channel 🚀